src/fu-tool.c

```diff
@@ -3067,6 +3073,48 @@ fu_util_switch_branch(FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_util_prompt_complete(priv->completion_flags, TRUE, error);
 }
 
+static gboolean
+fu_util_setup_console(GError **error)
```

Comment:
What do you think about putting this in common?  It can better prepare if some day we get daemon working on Windows too (dbus for Windows is a thing after all).